### PR TITLE
!isBrokenPR()

### PR DESCRIPTION
**Pull request by AltHit**
## About The Pull Request

Reupload.
Loading safe and tested fixes with separate PR to ensure that nothing needs reverting

## Why It's Good For The Game

Fixes runtimes and critical bugs
Also. New belt

## Testing

Already was tested on real Eris launch and proved to cause no issues

## Changelog
:cl:
add: Added multitool and pliers to robotist's toolbelt. Essential tools in fixing FBP's organs
del: Removes old and rusty toolbox from robotist on roundstart
tweak: tweaked a few things
balance: rebalanced something
fix: Fixed stakable items consumed whole while repairing organs
runtime: Removes runtime on attmpting healing certain eye organ wound and makes it actually cureable
spellcheck: fixed a typo in signals.dm comment
/:cl:

### CODE REVIEW

**Comment by Firefox13** [comment]
> Already was tested on real Eris launch and proved to cause no issues

Didn't it broke limb reattachment surgery? There *was* a reason it was quickly reverted.

**Comment by Firefox13** [comment]
> TL;DR: This is a working part of that PR, the other part will come separately later when I fix it.

Alright, just wanted to make sure as this wasn't clear from the PR's description

**Review by Firefox13 — APPROVED** [review]
